gvfs.h

```diff
@@ -12,6 +12,7 @@
  * The list of bits in the core_gvfs setting
  */
 #define GVFS_SKIP_SHA_ON_INDEX                      (1 << 0)
+#define GVFS_BLOCK_COMMANDS                         (1 << 1)
```

Comment:
I think we were skipping this bit for backwards-compatibility (users who only had the single-bit from a very old version). I suppose it has been long enough, but worth pausing to think if we can just use `(1 << 8)` instead.

Reply:
The single bits were ever only used for developer testing.  We never released a version that would set individual bits.  It always set all the bits by setting `core.gvfs` to `true`.